db/models.py

```diff
@@ -50,3 +59,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self) -> str:
+        return f"{self.created_at.date()} {self.created_at.time()}"
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        MovieSession, related_name="tickets", on_delete=models.CASCADE
+    )
+    order = models.ForeignKey(
+        Order, related_name="tickets", on_delete=models.CASCADE
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie.title} "
+                f"{self.movie_session.show_time.date()} "
+                f"{self.movie_session.show_time.time()} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        hall_rows = self.movie_session.cinema_hall.rows
+        hall_seats = self.movie_session.cinema_hall.seats_in_row
+        if not (1 <= self.row <= hall_rows):
+            raise ValidationError({
+                "row": [
+                    f"row number must be in available range: "
+                    f"(1, rows): (1, {hall_rows})"
+                ]
+            })
+        if not (1 <= self.seat <= hall_seats):
+            raise ValidationError({
+                "seat": [
+                    f"seat number must be in available range: "
+                    f"(1, seats_in_row): (1, {hall_seats})"
+                ]
+            })
+
+    def save(self,
+             force_insert: bool = False,
+             force_update: bool = False,
+             using: bool = None,
+             update_fields: bool = None
+             ) -> None:
```

Comment:
When you do not have any arguments on the line with a closing parenthesis, your open parenthesis should not have any arguments on the same line as well. Add this change to your code style.